include/fmt/ostream.h

```diff
@@ -189,6 +187,17 @@ struct fallback_formatter<T, Char, enable_if_t<is_streamable<T, Char>::value>>
   using basic_ostream_formatter<Char>::format;
 };
 
+inline void vprint_directly(std::ostream& os, string_view format_str,
+                            format_args args) {
+#ifdef _WIN32
+  auto buffer = memory_buffer();
+  detail::vformat_to(buffer, format_str, args);
+  detail::write_buffer(os, buffer);
+#else
+  ignore_unused(os, format_str, args);
+#endif
```

Comment:
Let's remove conditional compilation here. The only reason we use it in a similar case elsewhere is because of a separate compilation which is not the case here.

Reply:
Done.